Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/service/ConfigurationServiceException.java

```diff
@@ -55,7 +55,7 @@
     private static final long serialVersionUID = -1005166340298814306L;
 
     @SuppressWarnings("unused")
-    private ConfigurationFault fault = new ConfigurationFault();
+    private transient ConfigurationFault fault = new ConfigurationFault();
```

Comment:
It can and will be removed.  There aren't any references of it outside that class.